src/app/activate/page.tsx

```diff
@@ -103,7 +103,6 @@ const RegisterPage = () => {
         { message: "Veuillez renseigner une promo valide" },
       )
       .optional(),
-    score: z.number().int().min(0).max(4),
```

Comment:
We still want the score for activation, but we don't want it for login in, right ?